CrosswordMe.py

```diff
@@ -1,17 +1,51 @@
-#input
+from random import randint
+from random import sample
 
-words = []
-user_input = ""
+def print_puzzle(puzzle, line_count, column_count):
+    #Prints the puzzle puzzle on a board
+    for line in range(line_count):
+        for position in range(column_count):
+            print(str(puzzle[line][position]) + " ", end="") #Prints character in current position
+        print() #Prints Newline
 
-while 1 == 1:
-    user_input = input("Word: ")
-    if user_input.lower() == "done":
-        break
-    words.append(user_input)
+def word_input():
+    #Collects the words for the puzzle
+    words = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
+    user_input = ""
+    """while 1 == 1:
+        user_input = input("Word: ")
```

Comment:
Ok this is commented, but still:

infinite loops are always dangerous, how does the user know to type "done"?

maybbe try something like this:
```
>>> [word.strip() for word in input("gimme words: ").split(',')]
```